### PR TITLE
refactors and tweaks for rolling out a new couch-based backend

### PR DESCRIPTION
**Pull request by czue**
@snopoke 

some of the couch receiver tests aren't passing yet

### CODE REVIEW

File: prototype/config.py [hunk 1]
```diff
@@ -10,4 +10,6 @@
```
Comment (czue) [inline prototype/config.py:10]: (does this file do anything? it doesn't seem to in tests)

Comment (snopoke) [inline prototype/config.py:10]: No I don't think so, should have been deleted.

File: prototype/instance/testconfig.example.py [hunk 2]
```diff
@@ -3,3 +3,5 @@
 TESTING = True
 
 SQLALCHEMY_DATABASE_URI= 'postgresql://postgres:password@localhost/mobile_endpoint'
+
+COUCH_URI = 'http://localhost:5984'
```
Comment (snopoke) [inline prototype/instance/testconfig.example.py:7]: This should inherit from config.py. Same for localconfig.

File: prototype/config.py (hunk 1 above)
Comment (czue) [inline prototype/config.py:10]: cool, i can remove it on the next update

File: prototype/mobile_endpoint/backends/couch/models.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
+from couchdbkit import Document
+from mobile_endpoint.form.models import XFormInstance
+from mobile_endpoint.models import ToFromGeneric
+
+
+class CouchForm(Document, ToFromGeneric):
```
Comment (snopoke) [inline prototype/mobile_endpoint/backends/couch/models.py:6]: Is this necessary? Could you not just make the XFormInstance doc inherit from Document?

**Comment by snopoke** [comment]
:white_check_mark: LGTM, feel free to merge unless you want to keep working on it in the branch

File: prototype/mobile_endpoint/backends/couch/models.py (hunk 3 above)
Comment (czue) [inline prototype/mobile_endpoint/backends/couch/models.py:6]: i'll think about this, but will leave it for now

**Pull request merged**